vue/components/molecules/restrictions-alert/restrictions-alert.vue

```diff
@@ -10,11 +10,27 @@
                         {{ "ripe_commons.restrictions_alert.close_button" | locale }}
                     </a>
                 </div>
-                <div class="message-container">
-                    <span>
-                        {{ "ripe_commons.restrictions_alert.limited" | locale }}
-                        {{ "ripe_commons.restrictions_alert.back" | locale }}
+                <div
+                    class="message-container"
+                    v-bind:style="{ width: messages.length > 0 ? '100%' : null }"

```

Comment:
Why? If there are no messages this shouldn't even appear. Needing this trick may be a symptom that restrictionMessage shouldn't use this container.